integration/presto/src/main/prestodb/org/apache/carbondata/presto/impl/CarbonTableReader.java

```diff
@@ -281,7 +287,11 @@ private CarbonTableCacheModel getValidCacheBySchemaTableName(SchemaTableName sch
           createInputFormat(jobConf, carbonTable.getAbsoluteTableIdentifier(),
               new IndexFilter(carbonTable, filters, true), filteredPartitions);
       Job job = Job.getInstance(jobConf);
+      CarbonProperties.getInstance()
+          .addProperty(CarbonCommonConstants.IS_QUERY_FROM_PRESTO, "true");
```

Comment:
Can we move this initialization to a more common class..currently this will be set only in case of read

Reply:
@kunal642 , current carbon and presto integration is only in the query. Load or insert is not supported. 
So setting only in query flow should be enough I guess

Reply:
@ajantha-bhat If we have a common place, better to move there

Reply:
i think we can add only in connector classes. Moved to carbonDataModule. Please check if it is ok @ajantha-bhat @kunal642